src/scripts/utils/index.js

```diff
@@ -79,10 +79,15 @@ export function bindActions(dispatch) {
 
   result.loadFiscalDataPackage = function(url) {
```

Comment:
The codebase is supposed to support anything modelled by the `fiscalData` lib. FDP and CSV currently, with more in future. Therefore, we should not have here fdp-specific code.